.firebaserc

```diff
@@ -46,6 +48,13 @@
           "sparkle-1"
         ]
       }
+    },
+    "sparkle-2": {
+      "hosting": {
+        "sparkle-2": [
```

Comment:
I think this may have needed to be `sparkle-2a`, see https://github.com/sparkletown/sparkle/pull/996